Jenkinsfile

```diff
@@ -24,12 +24,13 @@ pipeline {
     quietPeriod(10)
   }
   triggers {
-    issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*')
+    issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?(?:module\\W+)?tests(?:\\W+please)?.*')
```

Comment:
even do the regexp is complimentary, I'd split it in two if it possible, to know that there are two types of comment trigger

Reply:
I'm afraid the `issueCommentTrigger` can be declared only once. I could potentially create two groups but the regex would be too long.
The proposed one supports the current use cases:
- `jennkins run the tests` and `jenkins run tests`
- `jenkins run the module tests for` and `jenkins run module tests for`

In any case, the filtering of running TAV is handled later on in the TAV stage.

what do you think if I leave it as it is? Or do you have another use case in mind?